docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -287,7 +287,7 @@ When DAGs are initialized with the ``access_control`` variable set, any usage of
     Breaking change
 
     Previously we were using two versions of UI, which were hard to maintain as we need to implement/update the same feature
-    in both versions. With this release we've removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
+    in both versions. With this release we have removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
     RBAC UI explicitly in the configuration now as this is the only default UI. We did it to avoid
     the huge maintenance burden of two independent user interfaces
 
```

Comment:
Question - is there any action that users need to take? Or is this more of an FYI doc? If there is any action, let folks know the steps here.

Reply:
God point - I added it. It's a bit different chang (the 've') change was mostly because spellcheck complained. I will make it separate change